composer.json

```diff
@@ -45,7 +45,8 @@
     },
     "autoload": {
         "psr-4": {
-            "Astrotomic\\Translatable\\": "src/Translatable/"
+            "Astrotomic\\Translatable\\": "src/Translatable/",
+            "Astrotomic\\Translation\\": "src/Translation/"
```

Comment:
There's no need for a new namespace. Just add the trait in the packages default namespace.